Zeplin.sketchplugin/Contents/Sketch/manifest.json

```diff
@@ -11,60 +11,90 @@
         "name": "Export Selected…",
         "identifier": "export",
         "shortcut": "cmd ctrl e",
-        "script": "export.cocoascript",
+        "script": "main.cocoascript",
         "icon": "Icons/icZeplinRunner.png",
         "description": "Export selected artboards and symbols to Zeplin."
     }, {
         "name": "Export Colors…",
         "identifier": "exportColors",
-        "script": "export.cocoascript",
+        "script": "main.cocoascript",
         "handler": "exportColors",
         "icon": "Icons/icZeplinRunner.png",
         "description": "Export colors to Zeplin."
     }, {
         "name": "Export Text Styles…",
         "identifier": "exportTextStyles",
-        "script": "export.cocoascript",
+        "script": "main.cocoascript",
         "handler": "exportTextStyles",
         "icon": "Icons/icZeplinRunner.png",
         "description": "Export text styles to Zeplin."
+    }, {
+        "name": "Artboards from Current Page…",
+        "identifier": "artboardsFromCurrentPage",
```

Comment:
Shouldn't we have the identifiers for the export items start with `export` prefix as well? i.e. `"exportArtboardsFromCurrentPage"`

Reply:
These ones are under the `Export All` menu therefore we didn't put `Export` word to he start of the names and thus I didn't put it to the identifier as prefix.